test/formatters/index.js

```diff
@@ -7,10 +7,49 @@ const options = { dateFormat, timeFormat };
 
 describe('formatters', () => {
   it('formatNumber', () => {
+    const data = {
+      stringVal: '21',
+      zeroedString: '00012',
+      floatString: '12.5',
+      number: 42,
+      smallNumber: 2,
+      largeNumber: 31415926535,
+      float: 61.66,
+      largeFloat: 3.1415926535,
+    };
+
     assert.equal(utils.formatNumber(1, 10), '0000000001', 'padded number correctly');
     assert.equal(utils.formatNumber(5), '05', 'sane defaults');
     assert.equal(utils.formatNumber(5000), '50', 'shortened number correctly');
     assert.equal(utils.formatNumber(5000, 7), '0005000', 'padded long number');
+
+    assert.isString(utils.formatNumber(data.number));
+    assert.isString(utils.formatNumber(data.smallNumber));
+    assert.isString(utils.formatNumber(data.smallNumber, 20));
```

Comment:
since you only check type conversion second argument check here does nothing. one type of argument here is enough, variations are tested properly below